os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/amebad/src/trace_uart.c

```diff
@@ -77,10 +77,10 @@ uint32_t traceuart_irq(void *data)
             break;
         case RUART_RECEIVE_LINE_STATUS:
             reg_val = (UART_LineStatusGet(TRACE_UART_DEV));
-            platform_debug("traceuart_irq: LSR interrupt, reg_val=%x\r\n",reg_val);
+            dbg("traceuart_irq: LSR interrupt, reg_val=%x\r\n", reg_val);
```

Comment:
`reg_val` is `uint32_t` type. Is it intended to print with `%x` format?

Reply:
Yes, to print it in Hex format.